articles/tutorials/advanced/2d_shaders/index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Advanced 2D Shaders in MonoGame
+description: A tutorial series on creating advanced 2D visual effects with custom shaders in MonoGame.
+---
+
+This tutorial series picks up where the "Building 2D Games" tutorial left off, diving deep into the world of custom shaders. The goal is to develop tooling for MonoGame to facilitate shader development, write some custom effects for _DungeonSlime_, and build intuition for how to approach shader programming.
+
+## What We Will Build
+
+Throughout this series, we will take our _Dungeon Slime_ game and add a whole new layer of visual polish. By the end, you will have implemented a variety of advanced shader effects, including:
+
+- A scene transition effect,
+- A color swap effect,
+- A 3d effect using `SpriteBatch`,
+- A lighting system using techniques from deferred rendering,
+- A shadow system using stencil buffers.
+
+![_Dungeon Slime_ will look like this at the end of this series](./videos/final.mp4)
+
+## Who This Is For
+
+This documentation is for intermediate MonoGame users. It assumes that you have a solid understanding of C# and have _already completed the entire "Building 2D Games" tutorial series._
+
+> [!IMPORTANT]
+> The concepts and code from the original tutorial, especially the "Shaders" chapter, are a mandatory prerequisite. We will be building directly on top of the final project from that series.
+
+## Table of Contents
+
+This documentation is organized to be read sequentially, as each chapter builds upon the techniques and code from the previous one.
+
+| Chapter                                                              | Summary                                                                                                                                  | Source Files                                                                                                                                       |
+| -------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Chapter 01: Introduction](01_introduction/index.md)                 | An overview of the advanced shader techniques covered in this tutorial series.                                                           | [Final Chapter from Building-2D-Games Series](https://github.com/MonoGame/MonoGame.Samples/tree/3.8.4/Tutorials/learn-monogame-2d/src/24-Shaders/) |
```

Comment:
Note, target for sample source should be the `3.8.5` branch, as that is the timeline for release